usecases/replica/coordinator.go

```diff
@@ -261,7 +263,7 @@ func (c *coordinator[T]) Pull(ctx context.Context,
 				// let's fallback to the backups in the retry queue
 				for hr := range hostRetryQueue {
 					resp, err := op(workerCtx, hr.host, isFullReadWorker)
-					if err == nil {
+					if err == nil || strings.Contains(err.Error(), ErrConflictExistOrDeleted.Error()) {
```

Comment:
I think error comparison here is not required as `op` is locally resolved on each node i.e. repair phase takes place after the coordinator finishes